CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contribution guide
+
+This guide aims to help developers working on this repository.
+
+## Changing all images without copy-paste
+
+We keep the multiple files for the different image versions in sync. Here is a
+tip on how to use `git` and `sed` to automatically apply patches targeting one
+image to all the others, without incurring in manual copy-paste.
+
+Supposing that changes were made to files in the 3.2 directory, we can then
+apply those changes to all other images:
+
+```bash
+for version in 2.4 2.6 3.0-upg; do
+  git diff -- 3.2 | sed "s|3.2/|${version}/|g" | git apply
```

Comment:
I use `--reject` a lot.  The point is that I get the `*.rej` files, and I open them by:
`vim affected/file.suffix{,.rej}` (or so..), `vsplit`, `b2`, ...

Anyway, I don't consider this is "no-cut-and-paste" approach.

Personally, I don't consider this auto-C&P much better than what we have now, because that is several seconds to do manually anyway.  Much worse is to